src/stdlib/SDL_malloc.c

```diff
@@ -5304,6 +5304,11 @@ mspace_mallopt(int param_number, int value)
 
 #endif /* !HAVE_MALLOC */
 
+#ifdef __NGAGE__
```

Comment:
Why is this needed? Does something need to be added to SDL_config_ngage.h instead?

Reply:
You're right; that's a remnant and doesn't belong there. I fixed it and the moved the `va_copy` definition as well. Thanks!